### PR TITLE
[FE-661] Feat/update v5 documentation

### PR DESCRIPTION
**Pull request by panvourtsis**

### CODE REVIEW

File: .storybook/addons.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import '@storybook/addon-docs/register';
+import { addons } from '@storybook/addons';
+// import '@storybook/addon-actions/register';
```
Comment (panvourtsis) [inline .storybook/addons.ts:3]: remove comments

File: .storybook/manager-head.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<link href="https://fonts.googleapis.com/css?family=Roboto:100,200,300,400,500&display=swap" rel="stylesheet" />
+<meta property="og:description" content="Ictinus design system by orfium.com" />
+<meta property="og:title" content="Ictinus Design System" />
+<title>Ictinus Design System</title>
```
Comment (panvourtsis) [inline .storybook/manager-head.html:4]: this doesnt work with webpack

File: docs/system/UTILS.stories.mdx [hunk 3]
```diff
@@ -105,8 +112,11 @@ It gets 2 params:
 2. shade
 
 And returns the AA color for that.
-   <br />
-   <br />
+
+{' '}
```
Comment (panvourtsis) [inline docs/system/UTILS.stories.mdx:116]: remove this

File: docs/tokens/globals/SPACING.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Proper spacing helps create visual hierarchy within a page and directs the user towards a desired action. Spacing in general uses
+multipliers of 4 for a consistent visual effect (with the exception of spacing.0 / spacing.1 / spacing.2 that are meant to cover small edge
+cases in components. All designs should exclusively use all spacing tokens provided here.
+
+## Tokens
+
+Spacing is part of [globals](?path=/docs/design-tokens-about--page#globals) tier of tokens.
+
+<SpacingSizingShowcase />
```
Comment (panvourtsis) [inline docs/tokens/globals/SPACING.stories.mdx:34]: spacing is broken

File: src/components/Avatar/Avatar.stories.mdx [hunk 5]
```diff
@@ -22,21 +22,36 @@ import { FIGMA_URL } from '../../utils/common';
 
 # Avatar
```
Comment (panvourtsis) [inline src/components/Avatar/Avatar.stories.mdx:23]: sectionheader

File: src/components/Avatar/AvatarStack.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+      {
+        type: 'figma',
+        name: 'Avatar',
+        url: `${FIGMA_URL}?node-id=3325%3A58246`,
+      },
+    ],
+  }}
+/>
+
+### Avatar Stack
```
Comment (panvourtsis) [inline src/components/Avatar/AvatarStack.stories.mdx:23]: section header

File: src/components/ExpandCollapse/ExpandCollapse.stories.mdx [hunk 7]
```diff
@@ -9,34 +9,34 @@ import { Fragment } from 'react';
 
 A minimally visual component that allows to expand and collapse content
 
-# Usage
+##
```
Comment (panvourtsis) [inline src/components/ExpandCollapse/ExpandCollapse.stories.mdx:12]: missing usage

File: src/components/RadioGroup/RadioGroup.stories.mdx [hunk 8]
```diff
@@ -13,43 +13,57 @@ If the `value` prop is defined it means the `RadioGroup` is controlled.
 If the `defaultValue` prop is defined it means the `RadioGroup` is not controlled.
 `RadioGroup` does **not** wrap the children in any DOM element, that's left as a user implementation
 
-# Usage
+##
```
Comment (panvourtsis) [inline src/components/RadioGroup/RadioGroup.stories.mdx:16]: missing usage

File: src/theme/globals/constants/typography.ts [hunk 9]
```diff
@@ -0,0 +1,168 @@
+const typography = {
```
Comment (panvourtsis) [inline src/theme/globals/constants/typography.ts:1]: remove this

File: .storybook/manager-head.html (hunk 2 above)
Comment (panvourtsis) [reply .storybook/manager-head.html:4]: it cannot be done till storybook v7

File: src/theme/globals/constants/typography.ts (hunk 9 above)
Comment (panvourtsis) [reply src/theme/globals/constants/typography.ts:1]: we are using this on `TypographyTokensShowcase`

File: src/components/ExpandCollapse/ExpandCollapse.stories.mdx (hunk 7 above)
Comment (panvourtsis) [reply src/components/ExpandCollapse/ExpandCollapse.stories.mdx]: fixed on https://github.com/Orfium/orfium-ictinus/pull/602/commits/b7f1359dc90fbc564437a368355fb1a515c28fd4

File: src/components/Avatar/AvatarStack.stories.mdx (hunk 6 above)
Comment (panvourtsis) [reply src/components/Avatar/AvatarStack.stories.mdx]: fixed on https://github.com/Orfium/orfium-ictinus/pull/602/commits/b7f1359dc90fbc564437a368355fb1a515c28fd4

File: src/components/Avatar/Avatar.stories.mdx (hunk 5 above)
Comment (panvourtsis) [reply src/components/Avatar/Avatar.stories.mdx]: fixed on https://github.com/Orfium/orfium-ictinus/pull/602/commits/b7f1359dc90fbc564437a368355fb1a515c28fd4

File: docs/system/UTILS.stories.mdx (hunk 3 above)
Comment (panvourtsis) [reply docs/system/UTILS.stories.mdx]: fixed on [`b7f1359` (#602)](https://github.com/Orfium/orfium-ictinus/pull/602/commits/b7f1359dc90fbc564437a368355fb1a515c28fd4)

File: src/components/DropdownButton/DropdownButton.stories.mdx [hunk 10]
```diff
@@ -27,27 +27,38 @@ import { FIGMA_URL } from '../../utils/common';
 
 # DropdownButton
```
Comment (kostasdano) [inline src/components/DropdownButton/DropdownButton.stories.mdx:28]: Maybe add SectionHeader here too

File: src/components/IconButton/IconButton.stories.mdx [hunk 11]
```diff
@@ -29,15 +29,15 @@ import { FIGMA_URL } from '../../utils/common';
```
Comment (kostasdano) [inline src/components/IconButton/IconButton.stories.mdx:29]: Maybe add SectionHeader here too

File: src/components/Typography/Typography.stories.mdx [hunk 12]
```diff
@@ -20,21 +19,34 @@ import { FIGMA_URL } from '../../utils/common';
 
 # Typography
```
[the remark this answers is not in the document]
Comment (panvourtsis) [reply src/components/Typography/Typography.stories.mdx]: fixed on [`94a0b8d` (#602)](https://github.com/Orfium/orfium-ictinus/pull/602/commits/94a0b8d04bf2195a0d76683cc48b2eb95a046be5)

File: src/components/IconButton/IconButton.stories.mdx (hunk 11 above)
Comment (panvourtsis) [reply src/components/IconButton/IconButton.stories.mdx:29]: fixed on [`94a0b8d` (#602)](https://github.com/Orfium/orfium-ictinus/pull/602/commits/94a0b8d04bf2195a0d76683cc48b2eb95a046be5)

File: src/components/DropdownButton/DropdownButton.stories.mdx (hunk 10 above)
Comment (panvourtsis) [reply src/components/DropdownButton/DropdownButton.stories.mdx]: fixed on [`94a0b8d` (#602)](https://github.com/Orfium/orfium-ictinus/pull/602/commits/94a0b8d04bf2195a0d76683cc48b2eb95a046be5)

**Review by kostasdano — APPROVED** [review]
LGTM 🎉 🚀

**Review by kostasdano — APPROVED** [review]
LGTM 🎉 🚀

**Pull request merged**